.eslintrc

```diff
@@ -67,7 +67,7 @@
       }
     },
     {
-      "files": ["config.js", "*.config.js", "main.js", "plopfile.js"],
+      "files": ["config.js", "*.config.js", "main.js", "plopfile.js", "src/**/*.js"],
```

Comment:
specifically to target `tier-1-definitions/breakpoints.js`. we could be more specific, but I kinda figure any `.js` file we're putting in `src/` is gonna be for some special purpose / will want this config